### PR TITLE
Site Design

### LINKED ISSUE #1: Site Design
Implement new site design

### PR DESCRIPTION
**Pull request by troym9731**
closes #1

### CODE REVIEW

File: public/scripts/vendor/npm.js [hunk 1]
```diff
@@ -1,13 +0,0 @@
-// This file is autogenerated via the `commonjs` Grunt task. You can require() this file in a CommonJS environment.
```
Comment (therealklanni) [inline public/scripts/vendor/npm.js]: Haha, no idea what this file was doing there.

File: server.js [hunk 2]
```diff
@@ -6,12 +6,15 @@ const debug = bug('SIR');
...
 import hbs from 'express-handlebars';
+import sassMiddleware from 'node-sass-middleware';
+
 import passport from 'passport';
 import session from 'express-session';
 import mongooseConnection from './lib/db/mongo'
 import connectMongo from 'connect-mongo'
 const MongoStore = connectMongo(session)
 
+import path from 'path';
```
Comment (therealklanni) [inline server.js]: Is this being used?

**Comment by therealklanni** [comment]
I'm going to go ahead and merge the first commit from this, to clean up the PR a bit (since it's a bunch of deletions, mostly).

**Comment by therealklanni** [comment]
Not sure why the PR didn't update after I merged in your first commit. :frowning:

File: server.js (hunk 2 above)
Comment (troym9731) [inline server.js]: Nope. It was before. I'll remove it.

**Pull request merged**